.changeset/loud-swans-repeat.md

```diff
@@ -0,0 +1,8 @@
+---
+'@primer/react-brand': patch
+---
+
+Style improvements to the `SubNav` component
+
+- A `SubNav.Link` with `aria-current` will now be underlined to indicate the current page
+- The vertical spacing around the `SubNav` component has been slightly reduced
```

Comment:
Out of interest, what was the reason for reducing the padding? The design was signed off previously, so has there been a recent change request? Asking as this will likely require more thorough dotcom testing and it increases the risk beyond what I assumed we're trying to address here (a11y remediation)

Reply:
Happy to add it back in, I thought it could be a visual bug as the Figma component appears to have a bit less vertical padding than we have in production.

To keep things simple I'll revert it

Reply:
Reverted and updated the snapshots to reflect it

Reply:
Oh cool, thanks for doing that. Makes sense to do that separately for audit trail reasons. Gives us more time to loop in Site FR and @danielguillan too, as we'd need them to confirm this change, and apply it to Figma also.